gulp-tasks/publish-lerna.js

```diff
@@ -6,20 +6,35 @@
   https://opensource.org/licenses/MIT.
 */
 
-const gulp = require('gulp');
+const execa = require('execa');
+const ol = require('common-tags').oneLine;
 
-const getNpmCmd = require('./utils/get-npm-cmd');
-const spawn = require('./utils/spawn-promise-wrapper');
+const logHelper = require('../infra/utils/log-helper');
 
-gulp.task('publish-lerna', () => {
-  return spawn(getNpmCmd(), [
-    'run', 'local-lerna',
-    '--',
-    'publish',
-    // Make sure to publish all packages, regardless of what's changed
-    '--force-publish',
+async function publish_lerna() {
+  const options = ['publish', '--force-publish'];
 
-    // The following flags can be used if publishing non-stable versions
-    // '--dist-tag=next',
-  ]);
-});
+  // gulp publish_lerna --distTag=blah takes precedence.
+  if (global.cliOptions.distTag) {
```

Comment:
This is one new bit of logic, around `lerna` dist tags. Both a CLI override, and then also defaulting to `next` if the current branch isn't `master`.

Reply:
+1 (really nice, thanks!)